src/dotnet/APIView/APIViewWeb/Managers/ReviewManager.cs

```diff
@@ -204,11 +204,12 @@ public async Task<ReviewModel> GetReviewAsync(ClaimsPrincipal user, string id)
             string name,
             string label,
             Stream fileStream,
+            string language = "",
```

Comment:
Input file type will be used when it's empty to find language processor to process input file.